src/main/webapp/WEB-INF/tags/errorPage.tag

```diff
@@ -20,9 +20,7 @@
 <body>
     <div class="navbar navbar-inverse navbar-fixed-top" role="navigation">
         <div class="container">
-            <div class="navbar-header">
-                <a class="navbar-brand" href="/index.jsp">TEAMMATES</a>
-            </div>
+            <t:navBarBrand/>
```

Comment:
Wrong place - `<t:navBarBrand/>` should only replace the `<a>` tag (and its contents). You're changing the structure by discarding `<div class="navbar-header">`.